code/graph_algorithms/src/Disjoint_Set_Union_algo.cpp

```diff
@@ -0,0 +1,55 @@
+#include<bits/stdc++.h>
+
+using namespace std;
+
+#define int long long int
+#define ld long double
+#define F first
+#define S second
+#define pb push_back
```

Comment:
Don't use these defines except for the first one; use `lli` not `int`

Reply:
basically for sake of using long long int i use int32_t instead it makes int as long long int so its easy and too applicable!